src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/læremidler/LæremidlerBeregnYtelseSteg.kt

```diff
@@ -75,6 +77,35 @@ class LæremidlerBeregnYtelseSteg(
     private fun beregnOgLagreInnvilgelse(
         vedtaksperioder: List<Vedtaksperiode>,
         saksbehandling: Saksbehandling,
+    ) {
+        val vedtaksperioderMedStatus =
+            when (saksbehandling.type) {
+                BehandlingType.FØRSTEGANGSBEHANDLING ->
+                    settStatusPåVedtaksperioder(
+                        vedtaksperioder = vedtaksperioder,
+                        vedtaksperioderForrigeBehandling = emptyList(),
+                    )
```

Comment:
Ved en førstegangsbehandling skal alle vedtaksperioder få status NY, trenger man egentligen kalle på `settStatusPåVedtaksperioder` då? Eller er det tilstrekkelig å kun returnere `vedtaksperioder` ?

Reply:
Det er strengt tatt tilstrekkelig å returnere `vedtaksperioder` som du sier. For flyt-forståelsen er jeg litt mer usikker. 

Uten  å kalle ´ settStatusPåVedtaksperioder´ kan det fort se ut som om disse vedtaksperiodene ikke skal ha status i det hele tatt, men det er jo ikke tilfellet.

Men dette er smak og behag, altså. Vi kan gjerne gjøre endringen hvis du tenker det er bedre. :)

Reply:
Ja, den metoden er ju også lagd for å kunne sende inn nullable vedtaksperioder, så den har ju på en måte støtte for å håndtere null vedtaksperioder. Det burde ev endres, eller at man gjør noe à la dette
```kotlin
val forrigeVedtaksperioder = saksbehandling.forrigeBehandlingId?.let { hentVedtak(it).data.vedtaksperioder}
        val vedtaksperioderMedStatus = settStatusPåVedtaksperioder(
            vedtaksperioder = vedtaksperioder,
            vedtaksperioderForrigeBehandling = forrigeVedtaksperioder,
        )
```

Der jeg tenker denne forenklet det

Reply:
Alle revurderinger har ikke `forrigeBehandlingId`

Reply:
> Alle revurderinger har ikke forrigeBehandlingId
Dævven, det var faktisk nytt for meg. Hvilke caser er det revurdering uten en forrige behandling?

Reply:
Lagde en oppgave på det